src/modules/app/includes/Plugin_App.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Wordlift\Modules\App;
+
+class Plugin_App {
+
+	private $handle = 'wl-angular-app';
+
+	public function register_handle() {
+		wp_register_script(
+			$this->handle,
+			plugin_dir_url( __DIR__ ) . 'js/app.js',
+			array(),
+			WORDLIFT_VERSION,
+			true
+		);
+
+		add_action(
+			'admin_enqueue_scripts',
+			function () {
+				// we dont want the filters to run if it was not enqueued.
+				if ( wp_script_is( $this->handle, 'enqueued' ) ) {
+					$settings = wp_json_encode( $this->get_settings() );
+					wp_add_inline_script( $this->handle, "var _wlPluginAppSettings = {$settings};" );
+				}
+			},
+			PHP_INT_MAX // This hook is running at last because we want to check if it was enqueued.
+		);
+	}
+
+	/**
+	 * @return mixed|null
+	 */
+	private function get_settings() {
+		return apply_filters(
+			'wl_plugin_app_settings',
+			array(
+				// Allows Stats Card to populate the settings.
+				'stats'           => apply_filters( 'wl_dashboard__stats__settings', array() ),
```

Comment:
@naveen17797 this is not good, again here you're making app aware of dashboard. Dashboard should hook to `wl_plugin_app_settings` and add the stats key along with its value. In no way app should be aware of the internal workings of wl_dashboard